code/obj/machinery/microwave.dm

```diff
@@ -17,7 +23,9 @@
 	var/operating = 0 // Is it on?
 	var/dirty = 0 // Does it need cleaning?
 	var/broken = 0 // How broken is it???
+	var/cook_time = 200 // The time to wait before spawning the item
 	var/list/available_recipes = list() // List of the recipes you can use
+	var/datum/recipe/cooked_recipe = null // The recipe being cooked
```

Comment:
code doc format

Reply:
As other comment, not sure what is wanted here.

Reply:
Ok, I think this is done correctly. Have tried replacing appropriate parts.